### PR TITLE
Add openshift ansible periodics

### PR DESCRIPTION
**Pull request by barbacbd**
** Adding periodics for the project in hopes that these can be tracked for regressions using the test grid.

### CODE REVIEW

**Comment by barbacbd** [comment]
/pj-rehearse periodic-ci-openshift-openshift-ansible-release-4.17-e2e-aws-workers-rhel8-nightly

**Comment by barbacbd** [comment]
/pj-rehearse periodic-ci-openshift-openshift-ansible-release-4.17-e2e-aws-workers-rhel8-nightly

**Comment by barbacbd** [comment]
/pj-rehearse periodic-ci-openshift-openshift-ansible-release-4.17-e2e-aws-workers-rhel8-nightly

File: ci-operator/config/openshift/openshift-ansible/openshift-openshift-ansible-master.yaml [hunk 1]
```diff
@@ -93,6 +93,21 @@ tests:
...
+      PLATFORM_VERSION: "8.4"
+    workflow: openshift-e2e-aws-workers-rhel
+- as: unit-nightly
+  commands: |
+    set -o pipefail
+    echo "${USER:-default}:x:$(id -u):$(id -g):Default User:$HOME:/sbin/nologin" >> /etc/passwd
+    tox
+  container:
+    from: test-bin
+  interval: 24h
```
Comment (r4f4) [inline ci-operator/config/openshift/openshift-ansible/openshift-openshift-ansible-master.yaml:110]: Unless each unit run is unique somehow, I think we can leave these as presubmits and drop them from this PR.

**Review by r4f4 — COMMENTED** [review]
/lgtm
But you might be missing steps if you want them to show up in the test grid: https://docs.ci.openshift.org/docs/how-tos/add-jobs-to-testgrid/#adding-a-non-release-gating-job-to-testgrid

**Pull request merged**